PowerValidatedSolutions.psm1

```diff
@@ -20714,6 +20721,384 @@ Function Get-PasswordPolicyConfig {
+                            if (-Not $checkReturn) {
+                                $encounterError = "True"
+                            }
+                        }
+                        "temporaryPassword"
+                        {
+                            $checkReturn = checkRange -name "$($product.Name):$($section.Name):temporaryPassword" -value $customConfig.($product.Name).($section.Name)."temporaryPassword" -minRange 1 -maxRange 99999 -required $true
+                            if (-Not $checkReturn) {
+                                $encounterError = "True"
+                            }
+                        }
+                        # Password Complexity section
+                        "policy"
+                        {	
+							$policyString = $customConfig.($product.Name).($section.Name)."policy"
+                            $customConfig.($product.Name).($section.Name)."policy" | Select-String -Pattern "^retry=(\d+)\s+min=(.+),(.+),(.+),(.+),(.+)" | Foreach-Object {$PasswdPolicyRetryValue, $PasswdPolicyMinValue1, $PasswdPolicyMinValue2, $PasswdPolicyMinValue3, $PasswdPolicyMinValue4, $PasswdPolicyMinValue5 = $_.Matches[0].Groups[1..6].Value}
+                            if ($PasswdPolicyRetryValue -eq "" -or $PasswdPolicyMinValue1 -eq "" -or $PasswdPolicyMinValue2 -eq "" -or $PasswdPolicyMinValue3 -eq "" -or $PasswdPolicyMinValue4 -eq "" -or $PasswdPolicyMinValue5 -eq "") {
+                                Write-LogMessage -Type ERROR -Message "The recommended policy setup should be retry=3 min=disabled,disabled,disabled,disbled,15"  -Colour Red
+								Write-LogMessage -Type ERROR -Message "The custom policy file shows $policyString"  -Colour Red
+                                $encounterError = "True"
+                            }
+                            if (($PasswdPolicyRetryValue -lt 0) -or ($PasswdPolicyRetryValue -gt 9999)) {
+                                Write-LogMessage -Type ERROR -Message "The recommended range for retry should be between 0 and 9999"  -Colour Red
+                                $encounterError = "True"
+                            }
+                            if ((($PasswdPolicyMinValue1 -lt 7) -or ($PasswdPolicyMinValue1 -gt 999)) -and ($PasswdPolicyMinValue1 -ine "disabled")) {
+                                Write-LogMessage -Type ERROR -Message "The recommended policy setup should be retry=3 min=disabled,disabled,disabled,disbled,15"  -Colour Red
+								Write-LogMessage -Type ERROR -Message "The custom policy file shows $policyString"  -Colour Red
+                                $encounterError = "True"
+                            }
+                            elseif ((($PasswdPolicyMinValue2 -lt 7) -or ($PasswdPolicyMinValue2 -gt 999)) -and ($PasswdPolicyMinValue2 -ine "disabled")) {
+                                Write-LogMessage -Type ERROR -Message "The recommended policy setup should be retry=3 min=disabled,disabled,disabled,disbled,15"  -Colour Red
```

Comment:
```suggestion
                                Write-LogMessage -Type ERROR -Message "The recommended policy configuration should be retry=3 min=disabled,disabled,disabled,disbled,15" -Colour Red
```

Reply:
resolved in 9bc7fcab7d319895f2ae65b7378a5d33841d60e4